### ISSUE TITLE
RESULT_CODE_HUNG in last update

### ISSUE THREAD

**Issue by fenevol** [QingWei-Li/notea#94]
Since last update, some notes hung the web browser with error RESULT_CODE_HUNG

Browser: Edge, Chrome (in Firefox it works OK).
Notea: 0.3.4 with minio

**Comment by QingWei-Li** [comment]
Have you tried other browsers?

**Comment by fenevol** [comment]
Yes, the problem is in Chrome and Edge but not in FIrefox. I haven't find a partner about the notes that hung the browsers.
